.github/workflows/ci.yml

```diff
@@ -62,14 +50,14 @@ jobs:
         env:
           NUGET_AUTH_TOKEN: ${{ steps.get_secrets.outputs.ado_pat }}
 
-      - run: ./Build.ps1 -SkipTests
```

Comment:
I believe we had two jobs and skipped tests because EphemeralMongo was not working on a specific Ubuntu distro. No longer a problem.